server/views/partials/additionalPssConditionsSummary.njk

```diff
@@ -56,7 +58,7 @@
 
     {% if additionalPssConditions.length > 0 %}
       {{ govukSummaryList({
-        attributes: { id: 'additional-pss-conditions-details' },
+        attributes: { id: 'additionalPssConditions' },
```

Comment:
Changed the ID to be the same as the field name being validated, so that the error summary links jump to the page anchor where the error is